wp-content/themes/chuongDemo/functions.php

```diff
@@ -0,0 +1,117 @@
+
+<?php
+// Add scripts and stylesheets
+function startwordpress_scripts() {
+
+	wp_enqueue_style( 'bootstrap', get_template_directory_uri() . '/css/bootstrap.min.css', array(), '3.3.6'  );
+	wp_enqueue_script( 'bootstrap', get_template_directory_uri() . '/js/bootstrap.min.js', array( 'jquery' ), '3.3.6', true );
+	wp_enqueue_script( 'my-great-script', 'https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js', array( 'jquery' ), '1.0.0', true );
+	wp_enqueue_style( 'blog', get_template_directory_uri() . '/css/blog.css' );
+}
+
+add_action( 'wp_enqueue_scripts', 'startwordpress_scripts' );
+// Add Google Fonts
+function startwordpress_google_fonts() {
+
+		wp_register_style('OpenSans', 'http://fonts.googleapis.com/css?family=Open+Sans:400,600,700,800');
+		wp_enqueue_style( 'OpenSans');
+}
+
+
+add_action('wp_print_styles', 'startwordpress_google_fonts');
+
+// WordPress Titles
+add_theme_support( 'title-tag' );
```

Comment:
you can add this feature in a setup function. Doc's https://developer.wordpress.org/themes/basics/theme-functions/#theme-setup